packages/ketcher-core/src/application/render/restruct/reatom.ts

```diff
@@ -564,6 +491,40 @@ class ReAtom extends ReObject {
       });
     }
   }
+
+  getLargestSectorFromNeighbors(struct: Struct): {
+    neighborAngle: number;
+    largestAngle: number;
+  } {
+    let angles: Array<number> = [];
+    this.a.neighbors.forEach((hbid) => {
+      const hb = struct.halfBonds.get(hbid);
+      hb && angles.push(hb.ang);
+    });
+    angles = angles.sort((a, b) => a - b);
+    const da: Array<number> = [];
+    for (let i = 0; i < angles.length - 1; ++i) {
+      da.push(angles[(i + 1) % angles.length] - angles[i]);
+    }
+    da.push(angles[0] - angles[angles.length - 1] + 2 * Math.PI);
+    let daMax = 0;
+    let ang = -Math.PI / 2;
+    for (let i = 0; i < angles.length; ++i) {
+      if (da[i] > daMax) {
+        daMax = da[i];
+        ang = angles[i];
+      }
+    }
```

Comment:
Please use full names for variables instead of 'hb', 'da', e.t.c

Reply:
Fixed